source/unified-test-format/unified-test-format.rst

```diff
@@ -1252,6 +1290,33 @@ expectedCmapEvent
   MUST be an empty document as all fields in this event are non-deterministic.
 
 
+expectedSdamEvent
+`````````````````
+
+The structure of this object is as follows:
+
+.. _expectedEvent_serverDescriptionChangedEvent:
+
+- ``serverDescriptionChangedEvent``: Optional object. Assertions for one or more
+  `ServerDescriptionChangedEvent <../server-discovery-and-monitoring/server-discovery-and-monitoring-monitoring.rst#events>`__ fields.
+
+  The structure of this object is as follows:
+
+  - ``previousDescription``: Optional object. A value corresponding to the server
+    description as it was before the change that triggered this event. 
+
+  - ``newDescription``: Optional object. A value corresponding to the server
+    description as it was after the change that triggered this event.
+
+  The structure of a server description object (which the ``previousDescription``
+  and ``newDescription`` fields contain) is as follows:
+
+  - ``type``: Optional string. The type of the server in the description. Test
+    runners MUST assert that the type in the published event matches this
+    value. See `SDAM: ServerType
+    <../server-discovery-and-monitoring/server-discovery-and-monitoring.rst#servertype>`__
+    for a list of valid values. 
```

Comment:
Ah good catch. These were originally in the spec but were removed after review, since they're not actually used in any tests curently. Removed from the schema.